packages/pyxis-react/scripts/generateIcons.ts

```diff
@@ -47,16 +47,15 @@ export const ${kebabToPascalCase(svgName)} = () => ${svgContent};
 
 const initialImport = `// DO NOT EDIT. This file is autogenerated by scripts/generateIcons.ts
 import React, { FC } from 'react';
-import { IconProps } from './types';
-import { Icon } from './Icon';
+import { Icon, IconProps} from './Icon';
```

Comment:
re-run the generate command
```suggestion
import { Icon, IconProps } from './Icon';
```